src/Blueprint/Blueprint/Models/PSBlueprintAssignment.cs

```diff
@@ -132,5 +126,33 @@ internal static PSBlueprintAssignment FromAssignment(Assignment assignment, stri
 
             return psAssignment;
         }
+
+        private static PSParameterValueBase GetAssignmentParameters(KeyValuePair<string, ParameterValueBase> parameterKvp)
+        {
+            PSParameterValueBase parameter = null;
+
+            if (parameterKvp.Value != null && parameterKvp.Value.GetType().Equals(typeof(ParameterValue)))
```

Comment:
@filizt the second check should be changed to `parameterKvp.Value is ParameterValue`

Reply:
Done.